.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,40 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '26 1 * * 2'
```

Comment:
Do we really need this, and if so why on Tuesdays specifically, since there's easily *at least* one PR opened/landed every week hence this analysis will run very frequently anyway?

Reply:
@Snuffleupagus that works for me. The date is generated automatically by GitHub so Thursday was just random.